org-formation/organization.yaml

```diff
@@ -566,6 +567,21 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         <<: !Include ./_platform_org_tags.yaml
 
+  GenieProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: genie-prod
+      RootEmail: aws-genie-prod@sagebase.org
+      Alias: org-sagebase-genie-prod
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: DPE
+        Project: Genie
+        AccountOwner: thomas.yu@sagebase.org
+        CostCenter: Genie-AACR BPC / 312100
+        budget-alarm-threshold: 1000
```

Comment:
RecoverDevAccount is set to 10K but this only set to 1K?

Reply:
Yes.  Since this account will only host one app', I think the threshold should be lower.